test/apps/opmon_python_publisher_test.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+from kafkaopmon.OpMonPublisher import OpMonPublisher
+from opmonlib.opmon_entry_pb2 import OpMonEntry 
+import getpass, time
+from druncschema.generic_pb2 import string_msg
+import click
+import logging
+
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--kafka-address', type=click.STRING, default="monkafka.cern.ch", help="address of the kafka broker")
+@click.option('--kafka-port', type=click.INT, default=30092, help='port of the kafka broker')
+@click.option('--topic', type=click.STRING, default="test.kafkaopmon")
```

Comment:
As a default value use `opmon_stream.test`. It's more consistent with the current topic name scheme. It's a minor thing really, but since there are a number of things to fix, we might as well sort this out.